src/studio/src/designer/frontend/packages/shared/src/pure/utils.ts

```diff
@@ -1,2 +1,7 @@
 export const isNumeric = (str: string) => parseInt(str).toString() === str;
 export const deepCopy = (value: any) => JSON.parse(JSON.stringify(value));
+export const removeKey = (obj: any, key: string) => {
+  const copy = deepCopy(obj);
```

Comment:
Trenger kanskje ikke `deepCopy` her siden `key` alltid vil være på det øverste nivået?
```suggestion
  const copy = { ...obj };
```

Reply:
Slik jeg har brukt denne så trenger jeg faktisk den andre delen av JSON-serialiseringen, den delen som fjerner metoder osv. Tror faktisk at spread-operatoren ikke funker på objekter som har metoder.

Reply:
Riktig: spread operator tar bare properties.
Metoder på objekter er litt nasty da, så det bør jo tas ut til funksjoner ASAP, nettopp fordi det kan føre til bugs i relasjon til denne kommentaren.

Reply:
Ja, men som i dette tilfellet så bruker jeg kode som vi ikke har kontroll på. Weird api det `useSearchParams`-apiet.